### ISSUE TITLE
Unmaintained Repo

### ISSUE THREAD

**Issue by mdstoll** [fbelavenuto/arpl#720]
With the release of DSM 7.2 RC and some other feedback like missing configurations like the DS1621xs+, I decided to move over to this fork of ARPL:
https://github.com/wjz304/arpl-i18n

Works solid and is up to date with the latest support. Hopefully this repo gets updated again later on but if you want to be bleeding edge, this fork seems to be the way to go.

**Comment by TineUser** [comment]
But with asian typeletters it isn't possible to read what changed etc.

**Comment by mdstoll** [comment]
Google Translate? 
I mean the guy maintaining this other repo is very clear in his release updates and what has changed. Nothing here has been updated since Feb 27th and the issues page is full with 180+ unanswered issues.

**Comment by carlesm** [comment]
I moved to this one:

https://github.com/jimmyGALLAND/arpl

Seems more mantained that the wjz304 one, with lots and lots of patches.

**Comment by mdstoll** [comment]
I see 2 releases without description (?)

https://github.com/jimmyGALLAND/arpl/releases

versus

https://github.com/wjz304/arpl-i18n/releases

**Comment by BobWs** [comment]
> I moved to this one:
> 
> https://github.com/jimmyGALLAND/arpl
> 
> Seems more mantained that the wjz304 one, with lots and lots of patches.

Tried your suggestion but I keep getting an error!

**Comment by TineUser** [comment]
I moved to https://github.com/AuxXxilium/arc. There's much more maintainance and very good support if having issues.

**Comment by mdstoll** [comment]
Interesting! That one is quite active as well indeed.

_What if for opensource code multiple developers would cooperate within the same repo. Just a thought xD_

**Comment by BobWs** [comment]
> What if for opensource code multiple developers would cooperate within the same repo. Just a thought xD

Agreed that would be much more efficient and one would only have a central place for questions and report issues.

**Comment by TineUser** [comment]
If I'm right, @wjz304 is cooperating with @AuxXxilium.
